### PR TITLE
TEAMNADO-3654: Add product details page

### PR DESCRIPTION
**Pull request by jlgre**

### CODE REVIEW

File: src/hooks/useSingleProduct.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useQuery, QueryObserverResult } from 'react-query';
+import { Product } from './useProducts';
+import Cookies from 'js-cookie';
+
+interface SingleProductApiData {
+  body: Product;
+}
+
+const fetchProductData = async (sku: string): Promise<Product> => {
+  const jwtToken = Cookies.get('cs_jwt');
```
[the remark this answers is not in the document]
Comment (jlgre) [reply src/hooks/useSingleProduct.ts]: Good call, updated

File: src/pages/DetailsPage/__tests__/DetailsPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      name: 'TEST Name',
+      productLine: 'TEST Line',
+      quantity: 3,
+      sku: 'TESTSKU',
+      serviceLevel: '',
+      serviceType: '',
+      unitOfMeasure: hasUom ? { name: 'test', quantity: 2 } : null
+    }
+  });
+
```
[the remark this answers is not in the document]
Comment (jlgre) [reply src/pages/DetailsPage/__tests__/DetailsPage.test.tsx:81]: Updated to make the null case test null everything

**Comment by cowanrc** [comment]
LGTM!

**Comment by jmbuscemi** [comment]
I'll look this over from a "requirements" stand point today

**Pull request merged**